repos/system_upgrade/common/actors/baculacheck/libraries/baculacheck.py

```diff
@@ -0,0 +1,50 @@
+from leapp import reporting
+from leapp.libraries.common.rpms import has_package
+from leapp.libraries.stdlib import api
+from leapp.models import InstalledRedHatSignedRPM
+
+# Summary for bacula-director report
+report_director_inst_summary = (
+    'Bacula director component will be upgraded. Since the new version is'
+    ' incompatible with the current version, it is necessary to proceed'
+    ' with additional steps for the complete upgrade of the Bacula backup'
+    ' database.'
+)
+
+report_director_inst_hint = (
+    'Back up your data before proceeding with the upgrade'
+    ' and use the command "/usr/libexec/bacula/update_bacula_tables <dbtype>" to upgrade'
+    ' the Bacula database after the system upgrade.'
+    ' The value of <dbtype> depends on the database backend, possible values are'
+    ' sqlite3, mysql, postgresql.'
+)
+
+
+def _report_director_installed():
+    """
+    Create report on bacula-director package installation detection.
+
+    Should remind user about present Bacula director package
+    installation and warn them about necessary additional steps.
+    """
+    reporting.create_report([
+        reporting.Title('bacula (bacula-director) has been detected on your system'),
+        reporting.Summary(report_director_inst_summary),
+        reporting.Severity(reporting.Severity.MEDIUM),
+        reporting.Tags([reporting.Tags.SERVICES]),
```

Comment:
@pirat89 I am wondering if time has come to introduce the brand new `Groups.POST` group here https://github.com/oamg/leapp/blob/master/leapp/reporting/__init__.py#L143 ? WDYT?

Reply:
that's a question. it's mixing work that is supposed to be done prior the upgrade and after the upgrade, however POST group should refer just to work supposed to be done after the upgrade.

Reply:
well it will be common to do work both before and after the upgrade: back up your stuff before in case the conversion goes wrong and convert it to an incompatible format after.

Reply:
@pirat89 I'd say we are not ready to introduce `Groups.POST` yet - we should first take care of our contributors' guidelines and explain, how we expect this group to be used.
So I'd leave this PR as is.